lti/templates/module_item_configurations/new_module_item_configuration.html.jinja2

```diff
@@ -1,13 +1,16 @@
 <script type="text/javascript">
   function handleSubmit(form) {
-    form.elements['url'].value = '{{ lti_launch_url }}' + encodeURIComponent(form.elements['url'].value);
+    form.elements['url'] && (form.elements['url'].value = form.elements['url'].value + encodeURIComponent(form.elements['document_url'].value));
```

Comment:
Does this value for sure show up in the post request made by the form?

Reply:
Yeah it does. :) even if we decide not to do react, it might be worth investigating a better way to do javascript that doesn't involve a bunch of script tags.